src/annotator/components/AdderToolbar.js

```diff
@@ -19,16 +19,16 @@ function ToolbarButton({ badgeCount, icon, label, onClick, shortcut }) {
 
   return (
     <button
-      className="annotator-adder-button"
+      className="AdderToolbar__button"
       onClick={onClick}
       aria-label={title}
       title={title}
     >
-      {icon && <SvgIcon name={icon} className="annotator-adder-button__icon" />}
+      {icon && <SvgIcon name={icon} className="AdderToolbar__icon" />}
```

Comment:
I simplified the naming down to `MainComponent__element` syntax which is conventional in our code rather than the unorthodox nested naming that we were using before.